packages/@lwc/integration-karma/test/component/dynamic-imports/index.spec.js

```diff
@@ -98,3 +104,51 @@ it('should not cache DOM elements', () => {
         });
     });
 });
+
+describe('slotted content', () => {
+    let consoleSpy;
+    beforeEach(() => {
+        consoleSpy = spyConsole();
+    });
+    afterEach(() => {
+        consoleSpy.reset();
+    });
+
+    it('reallocate slotted content after changing constructor', () => {
+        const elm = createElement('x-dynamic-slotted', { is: DynamicSlotted });
+        elm.ctor = ContainerFoo;
+
+        document.body.appendChild(elm);
+
+        expect(elm.shadowRoot.querySelector('[data-id="slot-default"]').assignedSlot).toBeDefined();
+        expect(elm.shadowRoot.querySelector('[data-id="slot-foo"]').assignedSlot).toBeDefined();
+        expect(elm.shadowRoot.querySelector('[data-id="slot-bar"]')?.assignedSlot).toBe(
```

Comment:
These `?.`s are causing a syntax error in Karma in IE11. I guess we don't transpile them. 😕

Reply:
Yes, this is it.